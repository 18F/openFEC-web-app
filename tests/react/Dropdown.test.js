const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const Dropdown = require('../../static/js/legal/filters/Dropdown');

it('renders dropdown', () => {
  const requestorOptions = [{value: 'opt1', text: 'Option 1'}, {value: 'opt2', text: 'Option 2'}];
  const component = rendered.create(
    <Dropdown key="ao_requestor_type" name="ao_requestor_type" label="Requestor Type"
    value="opt1" options={requestorOptions} handleChange={() => {}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
