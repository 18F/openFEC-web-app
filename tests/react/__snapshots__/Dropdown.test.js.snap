// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders dropdown 1`] = `
<div
  className="filter"
>
  <label
    className="label"
    htmlFor="ao_requestor_type"
  >
    Requestor Type
  </label>
  <select
    id="ao_requestor_type"
    name="ao_requestor_type"
    onChange={[Function]}
  >
    <option
      value="opt1"
    >
      Option 1
    </option>
    <option
      value="opt2"
    >
      Option 2
    </option>
  </select>
</div>
`;
